cf-templates/opensearch.yaml

```diff
@@ -519,3 +519,11 @@ Outputs:
           - /_dashboards
     Export:
       Name: AOSDashboardsPublicIP
+  AOSMasterPasswordSecretArn:
+    Value: !Ref AOSMasterPasswordSecret
+    Export:
+      Name: AOSMasterPasswordSecretArn
```

Comment:
Why was this output name pattern changed?

Reply:
I need ARN to KMS to pass to OSI.yaml CFT. OSI template is using lambda call to login into OpenSearch and update permissions for creating index and executing index management operations.